test/rest.test.js

```diff
@@ -1359,6 +1359,61 @@ describe('strong-remoting-rest', function() {
           });
       });
 
+      it('should allow customized xml root element', function(done) {
+        var method = givenSharedStaticMethod(
+          function bar(cb) {
+            cb(null, {a: 1, b: 2});
+          },
+          {
+            returns: { arg: 'data', type: 'object', root: true,
+            xml: { wrapperElement: 'foo'} },
+            http: { path: '/' }
+          }
+        );
+        request(app).get(method.classUrl)
+          .set('Accept', 'application/xml')
+          .set('Content-Type', 'application/json')
+          .send()
+          .expect('Content-Type', /xml/)
+          .expect(200, function(err, res) {
+            expect(res.text).to.equal(
+              '<?xml version=\"1.0\" encoding=\"UTF-8\"?>\n' +
+              '<foo>\n  ' +
+                '<a>1</a>\n  ' +
+                '<b>2</b>\n' +
+              '</foo>');
+            done(err, res);
+          });
+      });
+
+      it('should handle UTF-8 characters properly', function(done) {
```

Comment:
This test is testing not only UTF-8, but also special/reserved characters. Please update the test name to reflect that.